lib/broccoli/ember-app.js

```diff
@@ -368,7 +368,9 @@ class EmberApp {
 
     // Add testem.js to excludes for broccoli-asset-rev.
     // This will allow tests to run against the production builds.
-    if (this.options.fingerprint && this.options.fingerprint.exclude) {
+    if (this.isProduction || this.options.fingerprint) {
```

Comment:
I'm wondering why we only have this for `isProduction`. Might make sense to add that regardless of the build mode 🤔

Reply:
Ya, agree. @step2yeung any objection to always excluding?

Reply:
make sense, updated!